plasmapy/particles/particle_class.py

```diff
@@ -1465,7 +1465,7 @@ def electron_number(self) -> Integral:
             raise InvalidIonError(_category_errmsg(self, "ion"))
 
     @property
-    def isotopic_abundance(self) -> Real:
+    def isotopic_abundance(self) -> float:
```

Comment:
Similarly, the mypy cheat sheet recommends using `float` rather than `numbers.Real`.